app/code/Inchoo/Sample05/ViewModel/Event.php

```diff
@@ -33,23 +36,33 @@ class Event implements ArgumentInterface
      */
     protected $searchCriteriaBuilder;
 
+    /**
+     * Object manager
+     *
+     * @var ObjectManagerInterface
+     */
+    public $objectManager;
+
     /**
      * Event constructor.
      * @param RequestInterface $request
      * @param Registry $registry
      * @param EventRepositoryInterface $eventRepository
      * @param SearchCriteriaBuilder $searchCriteriaBuilder
+     * @param ObjectManagerInterface $objectManager
      */
     public function __construct(
         RequestInterface $request,
         Registry $registry,
         EventRepositoryInterface $eventRepository,
-        SearchCriteriaBuilder $searchCriteriaBuilder
+        SearchCriteriaBuilder $searchCriteriaBuilder,
+        ObjectManagerInterface $objectManager
     ) {
         $this->request = $request;
         $this->registry = $registry;
         $this->eventRepository = $eventRepository;
         $this->searchCriteriaBuilder = $searchCriteriaBuilder;
+        $this->objectManager = $objectManager;
```

Comment:
gdje si vidio ovo? Nemoj nikada direktno koristiti ObjectManager osim u iznimnim situacijama kao što je custom Factory klasa.